javascript-client/src/client.ts

```diff
@@ -168,8 +219,10 @@ export default class S3FileFieldClient {
    * @param multipartInfo Signed information returned from /upload-complete/.
    */
   protected async finalize(multipartInfo: MultipartInfo): Promise<string> {
-    const response = await this.api.post('finalize/', {
+    const response = await retry<AxiosResponse>(() => this.api.post('finalize/', {
```

Comment:
Again, do we know what happens when this is called repeatedly?

Reply:
Hopefully either 200 OK or 400 bad request. I don't know for sure.